buildlib/pr/io_demo/io-demo.yml

```diff
@@ -79,6 +109,9 @@ jobs:
             test_name: ${{ test.Key }}
             test_args: ${{ test.Value.args }}
             test_time: ${{ test.Value.duration }}
+            test_intefrafe: ${{ test.Value.interface }}
+            test_ucx_tls: ${{ test.Value.tls }}
+            test_display_name: ${{ test.Value.display_name }}
```

Comment:
for some reason doesn't work:
![image](https://user-images.githubusercontent.com/2255631/133614821-680cbd4a-9486-4976-8d99-b1797bd29379.png)